electron/config.json

```diff
@@ -1,72 +0,0 @@
-{
```

Comment:
Yep, I've moved this to our internal config repo given that nobody else cares what the config for our build of the electron app is.